src/backend/api/main.py

```diff
@@ -290,21 +294,27 @@ class EventDetailTypeConverter(BaseConverter):
     methods=["OPTIONS", "POST"],
     allow_headers=["Content-Type", "X-TBA-Auth-Id", "X-TBA-Auth-Sig"],
 )
-trusted_api.add_url_rule(
-    "/event/<string:event_key>/alliance_selections/update",
-    methods=["POST"],
-    view_func=update_event_alliances,
-),
-trusted_api.add_url_rule(
-    "/event/<string:event_key>/awards/update",
-    methods=["POST"],
-    view_func=update_event_awards,
-),
-trusted_api.add_url_rule(
-    "/event/<string:event_key>/info/update",
-    methods=["POST"],
-    view_func=update_event_info,
-),
+(
```

Comment:
why the extra nested parens?

Reply:
i have no idea how this occurred

Reply:
oh, there are trailing commas on the method calls, and vscode is formatting them as a tuple